### ISSUE TITLE
add support for standard DSN connect stirng

### ISSUE THREAD

**Issue by romaninsh** [atk4/data#230]
Allow Persistence::connect() to accept standard DSN string:

```
mysql://b66f427b1b479e:THTH@us-cdbr-iron-east-03.cleardb.net/heroku_fb6bd5aaa253862?reconnect=true
```

**Comment by DarkSide666** [comment]
for reference: http://php.net/manual/en/ref.pdo-mysql.connection.php

**Comment by romaninsh** [comment]
this has been implemented in 1.2 IIRC.
